com.reprezen.swagedit/src/com/reprezen/swagedit/editor/SwaggerDocument.java

```diff
@@ -148,8 +152,88 @@ public JsonNode getNodeForPath(String path) {
 	}
 
 	/**
-	 * Returns the yaml path of the element at the given line and column 
-	 * in the document.
+	 * Returns the region inside the document that can be reach through the
+	 * path.
+	 * 
+	 * @param path
+	 * @return region under the path
+	 */
+	public IRegion getRegion(String path) {
+		if (Strings.emptyToNull(path) == null)
```

Comment:
Inconsistent usage of the `{}` in the `if`s. You omit them here, but then use the 165-167. The code is good, but it makes it more difficult to read.